app/presenters/content_item_presenter.rb

```diff
@@ -1,5 +1,6 @@
 class ContentItemPresenter
   include Withdrawable
+  include BreadcrumbHelper
```

Comment:
Actually, shouldn't this be a module, like `Linkable` or `Widthdrawable`, rather than a helper?